### ISSUE TITLE
New Issue after #35

### ISSUE THREAD

**Issue by alperozoner** [ragnarok22/cryptobot_python#37]
cryptobot/_sync/client.py:42: CryptoBotError
=========================== short test summary info ============================
FAILED tests/test_sync_client.py::TestCryptoBotSyncClient::test_create_invoice
FAILED tests/test_sync_client.py::TestCryptoBotSyncClient::test_create_invoice_with_params
FAILED tests/test_sync_client.py::TestCryptoBotSyncClient::test_exchange_rates
FAILED tests/test_sync_client.py::TestCryptoBotSyncClient::test_get_balances
FAILED tests/test_sync_client.py::TestCryptoBotSyncClient::test_get_currencies
FAILED tests/test_sync_client.py::TestCryptoBotSyncClient::test_get_invoices
FAILED tests/test_sync_client.py::TestCryptoBotSyncClient::test_get_me - cryp...
========================= 7 failed, 2 passed in 2.94s ==========================
Error: Process completed with exit code 1.

**Comment by alperozoner** [comment]
Python tests / build (3.10) (pull_request): Failing after 24.

**Comment by ragnarok22** [comment]
Fixed: update exchanges rates. Thanks!
